folderAWG/OVON-Interoperability-Precis.md

```diff
@@ -0,0 +1,189 @@
+
+### IX	FREQUENTLY ASKED QUESTIONS
+
+Q: Why should I participate in the Open Voice Network Voice Agent Interoperability Initiative?
+
+A: More than 170 speech scientists, voice developers, and voice practitioners across 13 nations now contribute regularly to the work of the Open Voice Network. Broadly stated, these individuals believe that: 
+voice assistance will reach its optimum value worldwide for developers, enterprises, and voice platforms only if it is open, standards-based, and worthy of user trust;
+standards are of the greatest value when they are created through neutral, communal, and diverse research, debate, and development;
+standards drive and expand technology innovation and economic opportunity.
+
+Q: How does the Open Voice Network Voice Agent Interoperability Initiative (VAII) differ from other  voice interoperability initiatives? 
+ 
+A: The Open Voice Network applauds—and welcomes collaboration with—all initiatives that seek to expand the value of voice assistance through interoperability. In the simplest terms, the Open Voice Network seeks a future for voice that resembles today’s internet: an open, standards-based, voice-expanded and voice-enabled world-wide web, where billions of specific-purpose, unique-persona agents connect and share dialogs, data, context, and controls, and independent conversational capabilities abound.   
+
+To achieve this vision, we propose a software-centric, agent-to-agent approach to voice interoperability standardization, one pursued through communal debate and development that is platform-agnostic and technology-neutral.     
+
+There are now initiatives in the market that define interoperability in what might be described as a hardware-centric approach. This approach would allow conversational agents of different parentage to reside on a single device; users would toggle back and forth between agents, stopping and starting dialogues as needed. This is clearly useful in certain situations; indeed, the Open Voice Network expects to explore and develop standards for this functional (see enhancement #3, above.)  
+
+Q: Will the Open Voice Network develop an open conversational platform?
+
+A: No. The Open Voice Network will not develop an open, standards-based platform. 
+We seek to develop standards that will enable conversational agents of different platforms (open, proprietary, hybrid) to share dialogs, data, context, and controls. 
+
+Q: Will developers be able to insert Open Voice Network enhancements directly into their products?
+
+A: No. Enhancements will likely need to be refined and tailored to be effective in each vendor product as they use different platforms, languages, and implementation strategies. The true value of Open Voice Network interoperability work will most likely be found in the standard message formats, protocols, and data formats, and the algorithms used to implement each enhancement.  
+
+
+### IX.	ABOUT THE OPEN VOICE NETWORK (www.openvoicenetwork.org) 
+
+Vision
```

Comment:
A few of the sub headers in this section (e.g. "Vision", "Mission", etc) could use markup to make them easier to read. Banal thing.